io.catenax.quality_task/2.0.0/QualityTask.ttl

```diff
@@ -0,0 +1,184 @@
+
+:listOfQualityTasks a samm:Property ;
+   samm:preferredName "List of quality tasks"@en ;
+   samm:description "Property for the list of quality tasks."@en ;
+   samm:characteristic :ListOfQualityTasks .
+
+:metaInformation a samm:Property ;
+   samm:preferredName "Meta information"@en ;
+   samm:description "This meta information is valid for all entries in this data set. The meta information is used to tell the data consumer which selection criteria were applied by the data provider. "@en ;
+   samm:characteristic :MetaCharacteristic .
+
+:ListOfQualityTasks a samm-c:List ;
+   samm:preferredName "List of quality tasks"@en ;
+   samm:description "The list of quality tasks itself."@en ;
+   samm:dataType :SingleQualityTask .
+
+:MetaCharacteristic a samm:Characteristic ;
+   samm:preferredName "MetaCharacteristic"@en ;
+   samm:description "Chracteristic for the meta information."@en ;
+   samm:dataType :MetaInformation .
+
+:SingleQualityTask a samm:Entity ;
+   samm:preferredName "QualityTask"@en ;
+   samm:description "A quality task (qTask) defines why data is exchanged between 2 or more companies and what insights should be generated out of the transferred data. In addition there is a flag, what happens with the transferred data when this qTask is closed"@en ;
+   samm:properties ( [ samm:property :recordStatus; samm:optional true ] [ samm:property :creationDate; samm:optional true ] [ samm:property :component; samm:optional true ] [ samm:property :dataDeletion; samm:optional true ] [ samm:property :description; samm:optional true ] :qualityTaskId [ samm:property :status; samm:optional true ] [ samm:property :title; samm:optional true ] [ samm:property :listOfCompanies; samm:optional true ] ) .
+
+:MetaInformation a samm:Entity ;
+   samm:preferredName "Meta Information"@en ;
+   samm:description "This entity groups all meta information properties. Meta information properties give hints for a Catena-X data consumer to better understand, how this data set was created on the Catena-X data provider side."@en ;
+   samm:properties ( :selectionCriteria [ samm:property :selectionStart; samm:optional true ] [ samm:property :selectionEnd; samm:optional true ] ) .
+
+:recordStatus a samm:Property ;
+   samm:preferredName "Record status"@en ;
+   samm:description "The record operation enunumeration can be used to realize delta update concept.\nDelta updateupdate concept means:\n- You transfer an initial load of data\n- After the first week only the delta to the initial load is transferred\n\n\nThe record status describes whether this record is:\n- new=>This record is transferred the first time\n- update=> Some properties of this record have changed compared to a previous transfer\n- delete=> This record was transferred in the initial load or in a previous delta update, but is not \nused any more and therefore it should be deleted on data consumer side\n- same=> This record was transferred in the initial load or in a previous delta update"@en ;
```

Comment:
typo `enunumeration` , `updateupdate` ,` \nused`

Reply:
@TorstenHirsch looks like this one is skipped.

Reply:
fixed